python-avd/pyavd/_eos_designs/structured_config/base/platform_mixin.py

```diff
@@ -0,0 +1,129 @@
+        if self.shared_utils.is_sfe_interface_profile_supported and (sfe_member_interfaces_for_profile := self._get_sfe_interface_profile_member_interfaces):
+            default_sfe_interface_profile_name = "Default_Interface_Profile"
+            # build single ProfilesItem and append to Profiles
+            profiles_item = EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem(
+                name=default_sfe_interface_profile_name,
+                interfaces=sfe_member_interfaces_for_profile,
+            )
+            self.structured_config.platform.sfe.interface.profiles.append(profiles_item)
+            # indicate above profile as the one to apply for the device
+            self.structured_config.platform.sfe.interface.interface_profile = default_sfe_interface_profile_name
+
+    @cached_property
+    def _get_sfe_interface_profile_member_interfaces(
+        self: AvdStructuredConfigBaseProtocol,
+    ) -> EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.Interfaces | None:
+        """Returns list of eligible SFE interfaces with profile settings for structured config."""
+        # Iterate thru all L3 interfaces checking for those with 'rx_queue' config
+        # Also iterate thru member interfaces of all L3 Port-Channels with 'rx_queue' config
+        sfe_profile_member_interfaces = EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.Interfaces()
+        if not self.shared_utils.is_sfe_interface_profile_supported:
+            return sfe_profile_member_interfaces
+        # iterate thru each L3 interface
+        for interface in self.shared_utils.l3_interfaces:
+            intf_profile_settings = self._build_interface_with_rx_queue_settings(interface)
+            if intf_profile_settings is not None:
+                sfe_profile_member_interfaces.append(intf_profile_settings)
+
+        # iterate thru member interfaces of each L3 Port-Channel
+        for interface in self.shared_utils.node_config.l3_port_channels:
+            for member_intf in interface.member_interfaces:
+                intf_profile_settings = self._build_interface_with_rx_queue_settings(member_intf)
+                if intf_profile_settings is not None:
+                    sfe_profile_member_interfaces.append(intf_profile_settings)
+
+        if sfe_profile_member_interfaces:
+            return sfe_profile_member_interfaces
+        return None
+
+    def _build_interface_with_rx_queue_settings(
+        self: AvdStructuredConfigBaseProtocol,
+        l3_member_interface: EosDesigns._DynamicKeys.DynamicNodeTypesItem.NodeTypes.NodesItem.L3InterfacesItem
+        | EosDesigns._DynamicKeys.DynamicNodeTypesItem.NodeTypes.NodesItem.L3PortChannelsItem.MemberInterfacesItem,
+    ) -> EosCliConfigGen.Platform.Sfe.Interface.ProfilesItem.InterfacesItem | None:
+        """Returns one SFE interface with profile settings for structured config."""
+        if l3_member_interface._get("rx_queue"):
```

Comment:
We prefer returning fast + you should be able to use it like this:
```suggestion
        if not l3_member_interface.rx_queue:
            return
        [...] code
```